cli/common/tarantool.go

```diff
@@ -91,6 +92,31 @@ func GetNextMajorVersion(versionStr string) (string, error) {
 	return strconv.Itoa(major + 1), nil
 }
 
+func GetMajorCartridgeVersion(conn net.Conn) (int, error) {
```

Comment:
How is "scm" handled here?

Reply:
Also AFAIK "VERSION" generation it's quite new feature.

Reply:
But no, It's not a new feature. But "scm-1" is possible (https://github.com/tarantool/cartridge/blob/master/cartridge/VERSION.lua)

Reply:
Oh, thanks a lot! I will research.

Reply:
For 1.x `nil` is returned. In this case `1` is returned.
For `scm-1` version `2` is returned.